.github/workflows/test.yml

```diff
@@ -0,0 +1,104 @@
+name: WDIO tests + browserstack
+on:
+  push:
+    branches: [master, develop]
+  pull_request:
+    branches: [master, develop]
+
+# env:
+  # BASE_URL: https://deploy-preview-${{github.event.number}}--token-vega-xyz.netlify.app/
+jobs:
+  prepare:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: automation
+    # if: ${{ contains(github.event.pull_request.labels.*.name, 'readyTest') }}/
+    steps:
+        - name: Checkout
+          uses: actions/checkout@v2
+        - uses: actions/setup-node@v1
+          with:
+            node-version: 14.18.3
+        - name: Install root dependencies
+          run: npm run yarn:install:deps
+        - name: Install
+          run: npm install
+        # - name: Download wallet
+        #   run: curl -L https://github.com/vegaprotocol/vegawallet/releases/download/v${{ matrix.vegawallet-version }}/vegawallet-linux-amd64.zip -O
+        # - name: Unzip wallet
+        #   run: unzip ./vegawallet-linux-amd64.zip
+        - name: Initialize wallet
+          run: ./vegawallet init -f
+        # - name: List keys
+        #   run: ./vegawallet key list 
+        - name: Import config
+          run: ./vegawallet network import --from-url https://raw.githubusercontent.com/vegaprotocol/networks/master/fairground/fairground.toml
+        - name: List networks
+          run: ./vegawallet network list
+        - name: Start service
+          run: ./vegawallet service run --network fairground &
+        - name: 'BrowserStack Env Setup'  # Invokes the setup-env action
+          uses: browserstack/github-actions/setup-env@master
+          with:
+            username: ditmirhasani_8O6wBn
+            access-key: kapsvp3EvwLSmyDyJKxG
```

Comment:
These credentials should be stored as secrets

Reply:
done but will ask someone to help store secrets in github